src/frontend/screens/Settings/CreateOrJoinProject/CreateProject/index.tsx

```diff
@@ -130,7 +130,7 @@ export const CreateProject: NativeNavigationComponent<'CreateProject'> = ({
     if (!asset) return;
 
     // Only allow importing files with the desired extension
-    if (asset.name.endsWith('.mapeoconfig')) {
+    if (asset.name.endsWith('.mapeosettings')) {
```

Comment:
my understanding is that we're not supporting the old configurations (ending with `.mapeoconfig`) so I figured it made sense to only check for the new one instead of supporting both